packages/analytics-core/src/diagnostics/typings.ts

```diff
@@ -0,0 +1,14 @@
+export interface DiagnosticOmniMetrics {
+  metadata_type: string;
+  library: string;
+  accounting_time_min: number;
+  response_code: number;
+  trigger: string;
+  action: string;
+  event_count: number;
+}
+
+export interface DiagnosticEvent {
```

Comment:
I'm concerned with the naming `DiagnosticEvent` since this is not a real `BaseEvent`.

Can we use a different name e.g. `DiagnosticPayload` or `DiagnosticRequest`?

Reply:
Don't call it `event` to prevent customer confusion. Ping dx-pod [here](https://amplitude.slack.com/archives/C02E1H87E2C/p1697055738127019). Will use `DiagnosticRequest`.

Reply:
Also renamed `diagnosticEventBuilder` to `diagnosticRequestBuilder`